core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -1014,6 +1015,23 @@ public void syncPersistDirectory(InodeDirectory dir, JournalContext journalConte
             MkdirsOptions mkdirsOptions = MkdirsOptions.defaults().setCreateParent(false)
                 .setOwner(dir.getOwner()).setGroup(dir.getGroup()).setMode(new Mode(dir.getMode()));
             ufs.mkdirs(ufsUri, mkdirsOptions);
+            if (!ufs.mkdirs(ufsUri, mkdirsOptions)) {
```

Comment:
I think this should replace the line above.

Reply:
good catch, thanks